requirements.txt

```diff
@@ -5,3 +5,4 @@ pbr
 termcolor==1.1.0
 setproctitle
 ordereddict==1.1
+ujson
```

Comment:
This is why I think we should actually "only" focus on "virtualenv"(or --user) installations, system/package(rpm/deb/..) installations should come after.. (though we can keep it in mind).

because **any** system having a proper python2.6 or 2.7 installation (system wide or not) with pip and virtualenv correctly installed will (normally) be able to run alignak.. (if he has the eventual proper dev/headers files/libs required to build some eventual dependency)..